pkgs/development/python-modules/tinygrad/default.nix

```diff
@@ -96,11 +144,57 @@ buildPythonPackage rec {
     "test_vgg7"
   ];
 
-  disabledTestPaths = [
-    "test/extra/test_lr_scheduler.py"
-    "test/models/test_mnist.py"
-    "test/models/test_real_world.py"
-  ];
+  disabledTestPaths =
+    [
+      # Require internet access
+      "test/models/test_mnist.py"
+      "test/models/test_real_world.py"
+      "test/testextra/test_lr_scheduler.py"
+    ]
+    ++ lib.optionals (!testOpenclRuntime) [ ]
+    ++ lib.optionals (!rocmSupport) [ "extra/hip_gpu_driver/" ]
+    ++ lib.optionals (!cudaSupport) [ ];
+
+  # Require GPU access to run (not available in the sandbox)
+  pytestFlagsArray =
+    lib.optionals (!testCudaRuntime) [
+      "-k"
+      "'not TestCUDADevice'"
+    ]
+    ++ lib.optionals (!testRocmRuntime) [
+      "-k"
+      "'not TestHIPDevice'"
+    ]
```

Comment:
Can we add these items to `disabledTests`?
e.g.

```nix
disabledTests = [
  ...
] ++ lib.optionals (!testCudaRuntime) [
  "not TestCUDADevice"
] ++ ...
```

Reply:
In the end, those test do not exist at all.